src/components/structures/RoomView.js

```diff
@@ -375,6 +376,21 @@ module.exports = React.createClass({
         }
     },
 
+    onSearchContextFillRequest: function(backwards) {
+        if (!this.state.searchContext.getPaginateToken(backwards)) {
```

Comment:
Given `searchContext` can be `null` for a bunch of reasons, we really should be null-guarding accesses.